webcompat/form.py

```diff
@@ -233,14 +233,13 @@ def build_formdata(form_object):
     # Preparing the body
     body = u"""{metadata}
 **URL**: {url}
-**Browser / Version**: {browser}
-**Operating System**: {os}
-**Tested Another Browser**: {browser_test_type}
-
 **Problem type**: {problem_type}
 **Description**: {description}
-**Steps to Reproduce** {steps_reproduce}
 
+**Steps to Reproduce**: {steps_reproduce}
```

Comment:
@miketaylr hopefully the last commit & review :)
<img width="799" alt="screen shot 2017-07-17 at 1 34 25 pm" src="https://user-images.githubusercontent.com/562354/28266454-40ef2c82-6af5-11e7-9a2f-cf82b88b2a55.png">